src/main/generated/resources/data/minecolonies/researches/achievement.json

```diff
@@ -0,0 +1,4 @@
+{
+  "branch-name": "com.minecolonies.research.achievement.name",
+  "base-time": 0.0
```

Comment:
did you mean for this to be 1.0?

Reply:
Nope. This is the intent. Achievements don't have time, they should all be auto-researched and instant. This effectively enforces the instant part.

Reply:
I'm a bit confused. How is it a research then?

Reply:
It's using all of the conditional capabilities of the research system to unlock things. Specifically, if you have a sifter that is level 3, then the recipe for flint mesh becomes available automatically at the stonemason. Since this is a cross-crafter condition, the only system we have today that can handle that is the university.

Reply:
And these aren't visible in the uni GUI?

Reply:
They aren't visible until they unlock. Then they show that they've been completed.